e2e/framework/machineset.go

```diff
@@ -88,7 +88,7 @@ func CreateMachineSet(cl client.Client, params machineSetParams) *clusterv1.Mach
 		ms.Spec.Template.Spec.FailureDomain = &params.failureDomain
 	}
 
-	Expect(cl.Create(ctx, ms)).To(Succeed())
+	Expect(cl.Create(ctx, ms)).To(Succeed(), "Should have successfully created the MachineSet")
```

Comment:
can be more specific, 
```suggestion
	Expect(cl.Create(ctx, ms)).To(Succeed(), "Should have successfully created the CAPI MachineSet")
```